documentation/otn_redis_schema.md

```diff
@@ -204,13 +204,24 @@ sample-interval     = uint64
 
 ### AMPLIFIER
 
-*/openconfig-optical-amplifier*
+*;/openconfig-optical-amplifier:optical-amplifiers/amplifier/config*
+
+*;revision "2019-12-06" {reference "0.5.0"}*
 
 ```
-key                 = AMPLIFIER|<name>      ; string
-;field               = value
-name                = STRING
```

Comment:
No need the name field, as it is already part of the key. 
- duplicated value could cause data integrity issues. that is why openconfig key name is leafref to ../config/name
- simply the annotation and transformer (only can have empty annotation for ../amplifier/name)
- simple and generic xfmr: https://github.com/jjin62/sonic-mgmt-common/blob/master/translib/transformer/xfmr_oc_name.go

```
deviation /oc-opt-amp:optical-amplifier/oc-opt-amp:amplifiers/oc-opt-amp:amplifier {
      deviate add {
        sonic-ext:key-transformer "oc_name_key_xfmr";
      }
    }

    deviation /oc-opt-amp:optical-amplifier/oc-opt-amp:amplifiers/oc-opt-amp:amplifier/oc-opt-amp:name {
      deviate add {
      }
    }

    deviation /oc-opt-amp:optical-amplifier/oc-opt-amp:amplifiers/oc-opt-amp:amplifier/oc-opt-amp:config {
      deviate add {
      }
    }

    deviation /oc-opt-amp:optical-amplifier/oc-opt-amp:amplifiers/oc-opt-amp:amplifier/oc-opt-amp:config/oc-opt-amp:name {
      deviate add {
        sonic-ext:field-transformer "oc_name_field_xfmr";
      }
    }
```